packages/@guardian/source-react-components-development-kitchen/components/logo/Logo.tsx

```diff
@@ -47,31 +47,28 @@ const StandardLogo = () => {
-					fill="#FFE500"
-				/>
+			<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 116">
+				<g fill={brandText.primary}>
+					<path
+						data-content="The Guardian"
```

Comment:
Is `data-content` doing anything magical or is it basically a comment for devs to understand this `path` is for?